### PR TITLE
Reduce Fio logs verbosity

### PR DESCRIPTION
**Pull request by thehobbit85**

### CODE REVIEW

File: src/fio/fioEngine.js [hunk 1]
```diff
@@ -719,7 +719,7 @@ export class FioEngine extends CurrencyEngine {
     } catch (e) {
       // handle FIO API error
       if (e.errorCode && fioApiErrorCodes.indexOf(e.errorCode) > -1) {
-        this.log.error(
+        this.log(
```
Comment (paullinator) [inline src/fio/fioEngine.js:722]: This looks like an error in transaction creation which should not be silenced

File: src/fio/fioEngine.js [hunk 2]
```diff
@@ -736,7 +736,7 @@ export class FioEngine extends CurrencyEngine {
         }
         throw e
       } else {
-        this.log.error(
+        this.log(
```
Comment (paullinator) [inline src/fio/fioEngine.js:739]: This looks like an error in transaction creation which should not be silenced

**Review by paullinator — CHANGES_REQUESTED** [review]

**Comment by thehobbit85** [comment]
I agree that it's an error in transaction and it should not be silenced, but, there are some inconsistencies between the result type we expect from the server to what we actually get, which cause this error to be triggered far more than it should. I do plan to talk to Andrew about a more permanent fix, but I thought that this is good enough until the next release.

**Comment by thehobbit85** [comment]
And the comment I wrote applies for both logs

**Comment by thehobbit85** [comment]
Got a verbal approval from Paul

**Pull request merged**